src/creature.h

```diff
@@ -392,7 +392,7 @@ class Creature : virtual public Thing
 	Creature* attackedCreature = nullptr;
 	Creature* master = nullptr;
 	Creature* followCreature = nullptr;
-	std::list<Creature*> followedByCreatures;
+	std::vector<Creature*> followedByCreatures;
```

Comment:
Since you call the iteratee `follower`, you can call this `followers` for consistency

```suggestion
	std::vector<Creature*> followers;
```